pylint/checkers/refactoring.py

```diff
@@ -159,7 +159,12 @@ class RefactoringChecker(checkers.BaseTokenChecker):
                   'chained-comparison',
                   'This message is emitted when pylint encounters boolean operation like'
                   '"a < b and b < c", suggesting instead to refactor it to "a < b < c"',
-                  ),
+                 ),
+        'R1717': ('Consider using new dict initialization syntax',
+                  'consider-using-dict-comprehension',
+                  'Although there is nothing syntactically wrong with this code, '
+                  'it is hard to read and can be simplified to a dict comprehension',
```

Comment:
You can also mention the fact that it is faster since you don't create another transient list.